RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -34,6 +37,12 @@ class ProjectLoader {
      */
     private static Map<String, Analysis> _analysisCache = [:]
 
+    private static final int PSRC_UNSET = 0
```

Comment:
PSRC = Project Config Source? Maybe no abbrev.?
Also this is a typical task for an enum, isn't it?